src/GraphQL/Resolver/DomainContentResolver.php

```diff
@@ -198,6 +353,67 @@ public function resolveContentName(ContentInfo $contentInfo)
         return $this->repository->getContentService()->loadContentByContentInfo($contentInfo)->getName();
     }
 
+    private function getInputFieldValue($input, Repository\Values\ContentType\FieldDefinition $fieldDefinition)
+    {
+        $supportedInputTypes = ['ezstring', 'ezimage', 'eztext', 'ezrichtext', 'ezauthor'];
```

Comment:
Refactor this to `FieldValueInputMapper` objects (note: the default case at line 386 will handle quite a few field types thanks to `fromString()`.